subprojects/libnvme.wrap

```diff
@@ -1,6 +1,6 @@
 [wrap-git]
 url = https://github.com/linux-nvme/libnvme.git
-revision = ee30b6824f1209417e314d2c5b2e8d22b8707b6e
+revision = a849f9752edee2341feb52efe3d1bd9034f8ad38
```

Comment:
Please do a libnvme.wrap update as single commit first in the series. I'd like to avoid mixing code fixes with CI build changes. This makes backporting of changes simpler.

Reply:
splitted into a new commit for that !
as you commented